specification/network/resource-manager/Microsoft.Network/preview/2021-05-01-preview/examples/NetworkManagerPendingScopeConnectionSubscriptionList.json

```diff
@@ -0,0 +1,41 @@
+{
+  "title": "List Pending Azure Virtual Network Manager Scope Connection",
+  "parameters": {
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subscriptionA",
+    "body": {
+      "skipToken": "NextFakeSkipTokenCode"
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "ETag": "be3cfaa2-3d02-4624-92d7-a26766306c8b"
+      },
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subscriptionC/resourceGroup/rg1/providers/Microsoft.Network/networkManagers/testNetworkManager/scopeConnections/testScopeConnection",
+            "type": "Microsoft.Network/networkManagers/scopeConnections",
+            "name": "testScopeConnection",
+            "systemData": {
+              "createdBy": "b69a9388-9488-4534-b470-7ec6d41beef5",
+              "createdByType": "User",
+              "createdAt": "2021-01-11T18:52:27Z",
+              "lastModifiedBy": "b69a9388-9488-4534-b470-7ec6d41beef5",
+              "lastModifiedByType": "User",
+              "lastModifiedAt": "2021-01-11T18:52:27Z"
+            },
+            "properties": {
+              "tenantId": "6babcaad-604b-40ac-a9d7-9fd97c0b779f",
+              "connectionState": "Pending",
+              "resourceId": "subscriptions/f0dc2b34-dfad-40e4-83e0-2309fed8d00b",
```

Comment:
this should match the input parameter of subscriptionId

Reply:
good catch